api/composer.json

```diff
@@ -17,11 +17,11 @@
         "mll-lab/graphql-php-scalars": "^5.1",
         "mll-lab/laravel-graphql-playground": "^2.5",
         "nuwave/lighthouse": "^5.8",
-        "web-token/jwt-core": "^2.2"
+        "web-token/jwt-core": "^2.2",
+        "fakerphp/faker": "^1.9.1"
```

Comment:
Open for suggestions on this one.
Some seeders like Skill seeder required this and since its dev dependency these seeders wouldn't run in UAT ,prod env. and will cause miss the skills for ex.
This change required to populate skill & candidates in UAT env.